cli/browser-compatibility-table.js

```diff
@@ -0,0 +1,22 @@
+export function fixMdnURL(document) {
+  // Loop over, and mutate, all 'browser_compatibility' sections
```

Comment:
This comment isn't helping. It says what the code does, but not **why**. 
The why is that the BCD data comes from a library that is place where the URLs are always absolute. And in our case they actually map to URIs we want to make. Something like that.